src/main/java/com/synopsys/integration/detect/tool/signaturescanner/BlackDuckSignatureScanner.java

```diff
@@ -163,9 +164,14 @@ private void publishResults(SignatureScannerReport signatureScannerReport) {
                 SignatureScanPath scanPath = createScanPath(path, maxDepth, signatureScannerExclusionNamePatterns, providedExclusionPatterns);
                 signatureScanPaths.add(scanPath);
             }
-        } else if (dockerTarFile != null) {
-            SignatureScanPath scanPath = createScanPath(dockerTarFile.getCanonicalFile().toPath(), maxDepth, signatureScannerExclusionNamePatterns, providedExclusionPatterns);
-            signatureScanPaths.add(scanPath);
+        } else if (dockerTargetData.hasTarget()) {
```

Comment:
again, I'm not really sure the hasTarget() check adds value. Could be something I'm not seeing.

Reply:
@stevebillings I guess you're right, I just did it for aesthetic but it's unnecessary.